recipes/mlxtend/meta.yaml

```diff
@@ -0,0 +1,96 @@
+{% set name = "mlxtend" %}
+{% set version = "0.7.0" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "8c9206449d3e77ff52a288cd2741c10361bc39924b6101a777bda475b05e7f16" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
```

Comment:
It's more of what the `setup.py` actually needs to build the package. The `--single-version-externally-managed` makes it such that all you need to provide are external libraries that are imported by running `setup.py`.